src/ui/server/helpers/mappings/map-length-of-policy.ts

```diff
@@ -0,0 +1,24 @@
+import { DEFAULT } from '../../content-strings';
+import { COVER_PERIOD } from '../../constants';
+
+const { LESS_THAN_2_YEARS, MORE_THAN_2_YEARS } = COVER_PERIOD;
+
+/**
+ * mapLengthOfPolicyField
+ * maps length of policy based on database id
+ * @param {Number} answer
+ * @returns {String} Less or more than 2 years value
+ */
+const mapLengthOfPolicyField = (answer?: number) => {
+  if (answer === LESS_THAN_2_YEARS.DB_ID) {
+    return LESS_THAN_2_YEARS.VALUE;
+  }
+
+  if (answer === MORE_THAN_2_YEARS.DB_ID) {
+    return MORE_THAN_2_YEARS.VALUE;
+  }
+
+  return DEFAULT.EMPTY;
+};
+
+export default mapLengthOfPolicyField;
```

Comment:
Should this be different to the helper function in the API? `mapCoverPeriodField`

Reply:
This looks the same as `mapCoverPeriodField`, but ordered differently?

Can we rename this to be `CoverPeriod` instead of `LengthOfPolicy`, beacuse its consuming the cover period field/data "length of policy" is purely whats to the user in the summary list?